app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name">Clock</string>
-    <string name="tab1">Tab1</string>
-    <string name="tab2">Tab2</string>
-    <string name="tab3">Tab3</string>
+    <string name="Alarm">Alarm</string>
```

Comment:
Rename Alarm, Stopwatch, Timer and Clock string values:

1. Don't use Uppercase letters for resource names (ids, strings. dimens, etc)
2. Give more info about resource values - it is tab title, so name it as alarm_tab_title, timer_tab_title, etc